openfisca_core/scripts/__init__.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+
+import sys
+import importlib
+
+
+TAX_BENEFIT_SYSTEM_OPTIONS = {
+    'country_package': {
+        'short': 'c',
+        'help': 'country package to use. If not provided, an automatic detection will be attempted by scanning the python packages installed in your environment which name contains the word "openfisca".'
+        },
+    'extensions': {
+        'short': 'e',
+        'help': 'extensions to load',
+        'nargs': '*'
+        },
+    'reforms': {
+        'short': 'r',
+        'help': 'reforms to apply to the country package',
+        'nargs': '*'
+        }
+    }
+
+
+def add_tax_benefit_system_arguments(parser):
+    for option, properties in TAX_BENEFIT_SYSTEM_OPTIONS.iteritems():
+        parser.add_argument(
+            '-{}'.format(properties['short']),
+            '--{}'.format(option),
+            action = 'store',
+            help = properties['help'],
+            nargs = properties.get('nargs')
+            )
+
+    return parser
+
+
+def build_tax_benefit_sytem(country_package_name, extensions, reforms):
+    if country_package_name:
+        try:
+            country_package = importlib.import_module(country_package_name)
+            assert hasattr(country_package, 'CountryTaxBenefitSystem')
+        except:
+            print('ERROR: `{}` does not seem to be a valid Openfisca country package.'.format(country_package_name))
```

Comment:
The strongest way to use `print`, which leads to no runtime exception (`UnicodeDecodeError`) is:

```python
person_name = u'Félix' # Note the accent
print(u'Hello {}!'.format(person_name).encode('utf-8'))
```

(that's heavy, yeah)

In this case `country_package_name` can be only ASCII so it's OK, but it's implicit.